### DISCUSSION TITLE
转换格式时，总提示转换状态已经完成？

### DISCUSSION THREAD

**Discussion by xingelee** [zhongyang219/MusicPlayer2#466] [Q&A]
大虾：
软件太棒了，今天尝试了一下转换功能，发现有些小瑕疵，请继续完善哦：转换格式的时候，转换状态总是“完成”（实际上未开始），请教如何解决？3KS
282799887#qq.com

**Comment by zhongyang219** [comment]
这个貌似是转换失败了，转换状态应该显示“失败”的，显示“完成”应该是一个bug，目前正在调查。
